packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -123,41 +207,47 @@ export class TreeView extends FoundationElement {
         }
     };
 
+    private handleSelected(item: TreeItem): void {
+        if (this.currentSelected !== item) {
+            item.setAttribute("tabindex", "0");
+            if (this.currentSelected instanceof TreeItem && this.currentFocused) {
+                this.currentSelected.selected = false;
+                this.currentSelected.setAttribute("tabindex", "-1");
+            }
+            if (!this.currentSelected) {
+                this.slottedTreeItems.forEach((item: HTMLElement) => {
+                    if (item instanceof TreeItem) {
+                        item.setAttribute("tabindex", "-1");
+                    }
+                });
+            }
```

Comment:
When testing this, I found one subtle issue. If I select a node, and then I move to a *disabled* node and try to select it - the disabled node correctly **does NOT** receive selection, but the previous node *loses* selection. It seems like in the case of a disabled node, nothing should change from a selection state (the old node should retain selection).

Reply:
Should be fixed now, the tree item was preventing the selection event from firing and not entering the code path.